### DISCUSSION TITLE
Inverter not reachable after grid power outage

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#1398] [General]
### What happened?

My area had power outages and now that the grid is back on- i see that inverter is red and no way i can make it start.

Any help please?

log and screenshot attached.

RX Period End
16:08:28.581 > All missing
16:08:28.591 > Nothing received, resend whole request
16:08:28.599 > TX PowerControl Channel: 40 --> 51 84 24 45 92 80 19 67 00 81 02 00 D0 00 8B 
16:08:30.483 > RX Period End
16:08:30.496 > All missing
16:08:30.504 > Nothing received, resend whole request
16:08:30.515 > TX PowerControl Channel: 61 --> 51 84 24 45 92 80 19 67 00 81 02 00 D0 00 8B 
16:08:31.508 > PowerMeterClass: TotalPower: 189.44
16:08:31.609 > [DPL::announceStatus] inverter is offline (polling enabled? radio okay?)
16:08:31.642 > [    568.664] JK BMS: request for data sent
16:08:31.668 > [    568.676] JK BMS: busy waiting for or reading a message from the BMS
16:08:31.687 > [    568.693] JK BMS: a whole frame was received
16:08:31.700 > [    568.720] JK BMS: waiting for poll interval to elapse
![IMG_8362](https://github.com/user-attachments/assets/eae8e9d3-080c-43ec-a55f-395e3c6ccbb4)

### To Reproduce Bug

Power outage, then wen grid back inverter out not responding 

### Expected Behavior

Inverter works

### Install Method

Pre-Compiled binary from GitHub releases

### What git-hash/version of OpenDTU-OnBattery?

2024.06

### What firmware variant (PIO Environment)?

generic_esp32

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of OpenDTU-OnBattery, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.
- [X] I have double checked that my inverter does not contain a W in the model name (like HMS-xxxW) as they are not supported.

**Comment by Solarteur** [comment]
In my opinion, this is not a bug and not an error in openDTU-OnBattery: As you write, the inverter is flashing red, which means it has not restarted after the power failure.
Did you disconnect the inverter (again) and plug it back in after about 15 minutes?

Did you measure/check with a "load" whether there is mains voltage at the socket?

These tests only make sense when it is light in your area and the PV modules can produce electricity.

**Comment by schlimmchen** [comment]
Migrated to discussion, as this is not an OpenDTU-OnBattery issue, as already pointed out by @Solarteur.

**Comment by greymda** [comment]
sure.

nothing worked, except disconnecting all power from the inverter for a couple of minutes and then it started to respond.
worst thing, i had to again pre-charge capacitors when connecting cables back as it was tripping the breakers.
